spark-tpcds-benchmark-runner/var/conf/config.yml

```diff
@@ -33,5 +27,16 @@ spark:
   sparkConf:
     spark.yarn.jars: service/lib/*
     spark.sql.adaptive.enabled: true
-testDataDir: test_data
```

Comment:
didnt find the need for this since you can already specify this as part of the file system's baseUri